<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(31, 96, 131);</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>321</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="Name">
      <property name="styleSheet">
       <string notr="true">border:1px solid black;
font: 14pt &quot;MV Boli&quot;;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="Price">
      <property name="styleSheet">
       <string notr="true">border:1px solid black;
font: 14pt &quot;MV Boli&quot;;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="Count">
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;MV Boli&quot;;
border:1px solid black;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="SelectImageButton">
      <property name="styleSheet">
       <string notr="true">color: black;
background-color: rgb(249, 201, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>انتخاب عکس</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>200</y>
     <width>251</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">border:1.5px solid pink;
font: 16pt &quot;Impact&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>نام محصول:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">border:1.5px solid pink;
font: 16pt &quot;Impact&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>قیمت محصول:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">border:1.5px solid pink;
font: 16pt &quot;Impact&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>تعداد محصول:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">border:1.5px solid pink;
font: 16pt &quot;Impact&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>انتخاب فایل عکس:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>20</y>
     <width>171</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/[removal.ai]_1212fe80-07d2-470a-91eb-e306b0b6d4c8_20943859.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="selected_image_label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>510</y>
     <width>531</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ADD">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>440</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MV Boli&quot;;
background-color: rgb(255, 255, 127);</string>
   </property>
   <property name="text">
    <string>افزودن محصول</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
